pymc_marketing/clv/utils.py

```diff
@@ -56,12 +56,12 @@ def customer_lifetime_value(
     time: int, optional
         The lifetime expected for the user in months. Default: 12
     discount_rate: float, optional
-        The monthly adjusted discount rate. Default: 1
+        The monthly adjusted discount rate. Default: 0.01
     freq: string, optional
-        Frequency of discrete time steps used to estimate the customer lifetime value.
-        Defaults to "D" for daily. Other options are "W" (weekly), "M" (monthly), and "H" (hourly).
-        Smaller time frames estimate better the effects of discounting rate, at the cost of more
-        evaluations.
+        Sampling frequency of purchase history. Defaults to "D" for daily.
```

Comment:
Would units (of time) be more appropriate?

Reply:
Can you write the full suggestion?

Reply:
Units of purchase frequency. Defaults to "D" for daily.

^How does that sound?

Reply:
Pandas calls it "the frequency of the time series". I think the `freq` kwarg is borrowed directly from them 

https://pandas.pydata.org/docs/reference/api/pandas.DatetimeIndex.freq.html.

So maybe just borrow that directly?

Reply:
Yeah I think that sounds good!